crates/pink/src/runtime/pallet_pink.rs

```diff
@@ -19,11 +28,28 @@ pub mod pallet {
     }
 
     #[pallet::config]
-    pub trait Config: frame_system::Config {}
+    pub trait Config: frame_system::Config {
+        type Currency: Currency<Self::AccountId>;
+    }
 
     #[pallet::storage]
     pub(crate) type ClusterId<T: Config> = StorageValue<_, Vec<u8>, ValueQuery>;
 
+    #[pallet::storage]
+    #[pallet::getter(fn gas_price)]
+    pub(crate) type GasPrice<T: Config> = StorageValue<_, BalanceOf<T>, ValueQuery>;
```

Comment:
It can: https://github.com/kvinwang/js-sdk-mod/commit/4991740bae5443f22777ddc0062255b4a723586f#diff-1dd3d057805864fb5f9eb307e10cbbef61e9922d5c2a09c2407086dfab0ad707R5832